packages/react-native-saf-x/package.json

```diff
@@ -30,21 +32,23 @@
     "scoped",
     "storage",
     "SAF",
-    "storage-access-framework"
+    "storage-access-framework",
+    "fs"
   ],
+  "repository": "https://github.com/jd1378/react-native-saf-x",
   "author": "Javad Mnjd (https://github.com/jd1378)",
   "license": "AGPL-3.0-or-later",
   "homepage": "https://github.com/laurent22/joplin/tree/dev/packages/react-native-saf-x",
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
-    "@babel/core": "7.16.0",
-    "@types/react": "16.14.21",
-    "@types/react-native": "0.64.19",
-    "react": "18.2.0",
-    "react-native": "0.66.1",
-    "typescript": "4.9.4"
+    "@babel/core": "^7.17.9",
+    "@types/react": "^18.0.0",
+    "@types/react-native": "^0.70.6",
+    "react": "18.1.0",
+    "react-native": "0.70.6",
+    "typescript": "^4.9.4"
```

Comment:
If you modify package.json, make sure you commit yarn.lock too otherwise CI will fail. Also we only use pinned dependencies (so no `^`). This is so that we know exactly what we're using, and upgrades are managed by Renovate

Reply:
okay, done